ContactsApp/Contact.swift

```diff
@@ -2,14 +2,15 @@
 //  Contact.swift
 //  ContactsApp
 //
-//  Created by Davide Mendolia on 30/09/15.
-//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//  Created by Rohit Bansal on 04/10/15.
 //
 
 import Foundation
 
-protocol Contact {
-    var firstName: String { get }
-    var lastName: String { get }
-    var phonenumber: String { get }
+struct Contact : ContactProtocol {
```

Comment:
I do not quite get this point, just renamed -- Contact  to ContactProtocol, and NewContact to Contact.  Because, as for my understanding, ContactProtocol is the "base contract" and Contact is the base entity I would be using, in the same way as we were using NewContact (earlier).